testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -768,6 +768,102 @@ def test_warning(self, system, request):
         with pytest.warns(match="Some of the additional"):
             request.getfixturevalue(system)
 
+    def test_dump_reader_units_attribute(self):
+        """Test that DumpReader has proper units defined"""
+        from MDAnalysis.coordinates.LAMMPS import DumpReader
+
+        expected_units = {
+            "time": "fs",
+            "length": "Angstrom",
+            "velocity": "Angstrom/fs",
+            "force": "kcal/(mol*Angstrom)",
+        }
+
+        assert DumpReader.units == expected_units
+
+    def test_force_unit_conversion_factor(self):
```

Comment:
This is the only test that doesn't fail when the source patch in this PR is reverted.